src/main/java/at/ac/tuwien/kr/alpha/solver/heuristics/BerkMin.java

```diff
@@ -241,24 +232,16 @@ private int getMostActiveChoosableAtom(NoGood noGood) {
 	}
 	
 	private int getMostActiveChoosableAtom(Stream<Integer> streamOfLiterals) {
-		ActivityComparator activityComparator = new ActivityComparator();
-		Optional<Integer> mostActiveAtom = streamOfLiterals.map(Literals::atomOf).filter(this::isUnassigned).filter(isAtomActiveChoicePoint)
-				.max(activityComparator);
-		return mostActiveAtom.orElse(DEFAULT_CHOICE_ATOM);
+		return streamOfLiterals
+			.map(Literals::atomOf)
+			.filter(this::isUnassigned)
+			.filter(isAtomActiveChoicePoint)
+			.max(Comparator.comparingDouble(this::getActivity))
```

Comment:
`Comparator.comparingDouble` comes in very handy and makes `ActivityComparator` redundant.